src/views/tango/grid_widget.jsx

```diff
@@ -1,38 +1,101 @@
-import {GridWidget, gridSlice, gridStore} from "@waltz-controls/waltz-grid-widget";
+import {makeGridWidget} from "@waltz-controls/waltz-grid-widget";
 import React from "react";
 import ReactDOM from "react-dom";
 
 import {TangoDropTarget} from "@waltz-controls/waltz-webix-extensions";
 
 
-// const testDevice = {
-//     host: "localhost:10000",
-//     device: "test",
-//     attributes: [
-//         {
-//             name: "double_scalar",
-//             value: 249.43882402802603
-//         }
-//     ],
-//     commands: []
-// }
-
 const grid_widget = webix.protoUI({
     name: 'grid_widget',
 
     defaults:{
         borderless:true
     },
 
-    addDevice(device){
```

Comment:
This will break dnd functionality as this method is responsible for adding new devices